otri/filtering/filters/phase_filter.py

```diff
@@ -1,24 +1,28 @@
-from ..filter import Filter, Stream, Collection
-from typing import Mapping, Callable, Any
+from ..filter import Filter, Stream, Sequence, Mapping, Any
+from typing import Callable, Collection
 
 
 class PhaseFilter(Filter):
     '''
     Applies an operation to the i atom's keys and i+c atom's keys.
+
     Input:
         Single stream ordered by datetime
     Output:
-        A single Stream containing n - c atoms, whose fields are the
+        Single stream containing n - c atoms, whose fields are the
         result of the given operation between the atoms at positions
-        i and i+c in the input Stream. Such fields are the only fields.
+        i and i+c in the input Stream. Such fields are the only fields
+        of the output data.
     '''
 
-    def __init__(self, input_stream: Stream, keys_to_change: Mapping[str, Callable[[Any, Any], Any]], distance: int):
+    def __init__(self, input: str, output: str, keys_to_change: Mapping[str, Callable[[Any, Any], Any]], distance: int):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)